.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+X. Enhancement - Added Sywell (EGBK) SMR - thanks to @JackG2004 (Jack Gil)
```

Comment:
Might want to add yourself?

Reply:
not mine - was showing him how to use github